File: January 6, 2026.py
def vowel_case(s):

    vowels = ["a", "e", "i", "o", "u"]
    new_s = ""

    for char in s:
        if char.lower() in vowels:
            new_s += char.upper()
        else:
            new_s += char.lower()

    return new_s

File: January 6, 2026.js
function vowelCase(str) {

  const vowels = ["a", "e", "i", "o", "u"];

  const newStr = str.split("").map((char) => {
    if(vowels.includes(char.toLowerCase())) {
      return char.toUpperCase();
    } else {
      return char.toLowerCase();
    }
  }).join("");

  return newStr;
}

vowelCase("vowelcase")